.pre-commit-config.yaml

```
@@ -43,3 +43,13 @@ repos:
     - flake8-2020>=1.6.0
     - flake8-docstrings
     - flake8-pytest-style>=1.0.0
+- repo: local
+  hooks:
+  - id: docs-rebuild
+    name: Checks that generated docs are in-sync with commit
+    description: >-
+      Regenerate rules documentation from source and report failure if the
+      the the result is modified.
+    entry: sh -c "pip install . && python -m ansiblelint -L -f rst > docs/docsite/rst/rules/default_rules.rst"
```

can we multiline this?
```suggestion
    entry: >-
      sh -c "
        pip install . &&
        python -m ansiblelint -L -f rst
        >
        docs/docsite/rst/rules/default_rules.rst
      "
```